packages/feedback/src/modal/components/Dialog.css.ts

```diff
@@ -133,6 +133,11 @@ export function createDialogStyles(): HTMLStyleElement {
   padding: 6px 12px;
 }
 
+.form__input::placeholder {
+  color: var(--input-foreground);
+  opacity: 0.65;
```

Comment:
I tried my best to I tried my best to find an opacity that kept the color the same as before

Reply:
Unfortunately this probably needs a new configuration option, e.g. `inputPlaceholder` or `inputPlaceholder`, I'm not positive that opacity will satisfy all color schemes

Reply:
<img src="https://media4.giphy.com/media/gfsQffBnuc6e096brx/giphy.gif"/>
yea it works well with black and white, but there are prob some color schemes where this will look wonky :( I would like to avoid adding another configuration option if possible, maybe we could ask @Jesse-Box to weigh in?

Reply:
Aligned with Jesse and decided to just use opacity for placeholder text because it's a safe assumptions that people would use white/gray/black colours for their inputs and custom colour text is usually reserved for buttons and links